docs/hypernode-platform/ftp/how-to-configure-ftp-sftp-on-hypernode.md

```diff
@@ -25,7 +25,22 @@ File Transfer Protocol (FTP) and Secure File Transfer Protocol (SFTP) are both w
 
 NB: We strongly recommend using SSH or SFTP for file transfers. FTP is an inherently unsafe protocol, and should only be used for syncing data with suppliers that can’t be done through the Magento API.
 
-## Managing FTP
+## Managing FTP Users in Control Panel
+
+To manage FTP users in the Control Panel, follow these steps:
+
+1. **Navigate to Hypernode:** Go to the Hypernode you want to manage FTP accounts for. In the side menu, locate and click on the "FTP" menu item.
+1. **View Existing Users:** If you already have existing users created via the CLI tool, they will be listed on this page. If no FTP users exist, you will see a form to create a new one.
+1. **Fill Out the Form:** In the form, you can specify the FTP username, password, and home directory for a new user.
+1. **Delete Existing User:** In the user list, you can click the "trash" icon in the row to delete the user.
+
+**Important Points to Note:**
```

Comment:
More semantic to wrap this block in
````
```{admonition} Important points to note
:class: tip

- If you create a new user with an existing username, the existing user will be overwritten with the new password and home directory.
- Home directories should start with '/' and must exist on your Hypernode.
- The same rules apply to the CLI tool, and these instructions are equivalent.
```
````

[Read more about admonitions here](https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html)

Reply:
So you can do things like 
<img width="792" alt="image" src="https://github.com/ByteInternet/hypernode-docs-next/assets/1165302/c861bc57-ebe8-4050-a001-96386fbf7774">

Reply:
@tdgroot great, it looks better. can you merge it we are not allowed to merge to the base branch.


    https://docs-ephj21c3q.hypernode.io/hypernode-platform/ftp/how-to-configure-ftp-sftp-on-hypernode.html


<img width="793" alt="afbeelding" src="https://github.com/ByteInternet/hypernode-docs-next/assets/13779866/451d1209-d00f-402c-8813-88f0d1941cea">